woocommerce/woocommerce-android
# Extract JITM analytics into a separate class

Closes: #7668 


### Description

This PR extracts all the analytics related JITM into a separate class `JitmTracker`.

The reason behind this refactoring is not to clutter `MyStoreViewModel` with too much JITM-related analytics logic.

### Testing instructions


Ensure CI is happy

Smoke test JITM and ensure tracks are being logged as expected.
